esp-box/idf_component.yml

```
@@ -1,13 +1,13 @@
 
-version: "2.4.2"
+version: "2.5.0"
 description: Board Support Package for ESP-BOX
 url: https://github.com/espressif/esp-bsp/tree/master/esp-box
 
 targets:
   - esp32s3
 
 dependencies:
-  idf: ">=5.0"
+  idf: ">=4.4"
```

According to https://github.com/espressif/esp-bsp/pull/192#issuecomment-1643322561, you should change this to 4.4.5:
```suggestion
  idf: ">=4.4.5"
```

---

Sure. I think, there should be one CI test for exactly IDF 4.4.5 and second one for latest release IDF 4.4 
Are you agree?

---

This is not related to CI test, i think... we can keep CI running for release/v4.4, just need to make sure that this component won't be installed if IDF version is <= 4.4.4 — preventing the issue which @donghengqaz ran into.